compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -2065,6 +2071,34 @@ class ShapeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::NodeS
     return loco::TensorShape{max_output_size_value};
   }
 
+  loco::NodeShape visit(const luci::CircleNonMaxSuppressionV5Out *node) final
+  {
+    const loco::DataType S32 = loco::DataType::S32;
+
+    auto nmsv5 = dynamic_cast<const luci::CircleNonMaxSuppressionV5 *>(node->input());
+    if (nmsv5 == nullptr)
+      INTERNAL_EXN("CircleNonMaxSuppressionV5 IR is not configured correctly");
+
+    auto index = node->index();
+    if (index == 2)
+      return loco::TensorShape({0});
```

Comment:
Could you provide a reference for this?

Reply:
Ouput `num_selected_indices` is a scalar, maybe I've used wrong 0D tensor shape representation here?
Here's how it's shape is defined in tflite kernel:
https://github.com/tensorflow/tensorflow/blob/2a96849f478627ad03b8f20f8c71e85d6d7480e3/tensorflow/lite/kernels/non_max_suppression.cc#L58

Reply:
Thanks for the reference. I think current code is OK.